docs/ops.html

```diff
@@ -346,6 +346,127 @@
+Configs for brokers '2' are leader.replication.throttled.rate=1000000,follower.replication.throttled.rate=1000000
+Configs for brokers '1' are leader.replication.throttled.rate=1000000,follower.replication.throttled.rate=1000000</pre>
+
+<p>This shows the throttle applied to both leader and follower side of the replication protocol. By default both sides
+    are assigned the same throttled throughput value. </p>
+
+<p>To view the list of throttled replicas:</p>
+
+<pre>$ bin/kafka-configs.sh --describe --zookeeper localhost:2181 --entity-type topics
+Configs for topic 'my-topic' are leader.replication.throttled.replicas=1:102,0:101,
+    follower.replication.throttled.replicas=1:101,0:102</pre>
+
+<p>Here we see the leader throttle is applied to partition 1 on broker 102 and partition 0 on broker 101. Likewise the
+    follower throttle is applied to partition 1 on
+    broker 101 and partition 0 on broker 102. </p>
+
+<p>By default kaka-reassign-partitions.sh will apply the leader throttle to all replicas that exist before the
+    rebalance, any one of which might be leader.
+    It will apply the follower throttle to all move destinations. So if there is a partition with replicas on brokers
+    101,102, being reassigned to 102,103, a leader throttle,
+    for that partition, would be applied to 101,102 and a follower throttle would be applied to 103 only. </p>
+
+
+<p>If required, you can also use the --alter switch on kafka-configs.sh to alter the throttle configurations manually.
+</p>
+
+<h5>Safe usage of throttled replication</h5>
+
+<p>Some care should be taken when using throttled replication. In particular:</p>
+
+<p><i>(1) Throttle Removal:</i></p>
+The throttle should be removed in a timely manner once reassignment completes (by running kaka-reassign-partitions
+—verify).
+
+<p><i>(2) Ensuring Progress:</i></p>
+<p>If the throttle is set too low, in comparison to the incoming write rate, it is possible for replication to not
+    make progress. This occurs when:</p>
+<pre>max(BytesInPerSec) > throttle</pre>
+<p>
+    Where BytesInPerSec is the metric that monitors the write throughput of producers into each broker. </p>
+<p>The administrator can monitor whether replication is making progress, during the rebalance, using the metric:</p>
+
+<pre>kafka.server:type=FetcherLagMetrics,name=ConsumerLag,clientId=([-.\w]+),topic=([-.\w]+),partition=([0-9]+)</pre>
+
+<p>The lag should constantly decrease during replication.  If the metric does not decrease the administrator should
+    increase the
+    throttle throughput as described above. </p>
+
+<p><i>(3) Avoiding long delays during replication:</i></p>
+<p>The throttled throughput should be large enough that replicas cannot be starved for extended periods. A good,
+    conservative rule of thumb is
+    to keep throttle above #brokers MB/s where #brokers is the number of brokers in your cluster.  </p>
```

Comment:
Hmm, if there are 100 brokers in the cluster, does that mean the min throttle value has to be 100MB/sec?